Tests/Tests/PayWithLinkViewController-WalletViewModelTests.swift

```diff
@@ -106,10 +106,15 @@ class PayWithLinkViewController_WalletViewModelTests: XCTestCase {
 extension PayWithLinkViewController_WalletViewModelTests {
 
     func makeSUT() throws -> PayWithLinkViewController.WalletViewModel {
+        let paymentIntentJson = try XCTUnwrap(STPTestUtils.jsonNamed(STPTestJSONPaymentIntent))
+        let orderedPaymentJson = ["card", "link"]
+        let paymentIntentResponse = ["payment_intent": paymentIntentJson,
+                                     "ordered_payment_method_types": orderedPaymentJson] as [String : Any]
+        let linkSettingsJson = ["link_funding_sources": ["CARD"]]
+        let response = ["payment_method_preference": paymentIntentResponse,
+                        "link_settings": linkSettingsJson]
         let paymentIntent = try XCTUnwrap(
-            STPPaymentIntent.decodedObject(
-                fromAPIResponse: STPTestUtils.jsonNamed(STPTestJSONPaymentIntent)
-            )
+            STPPaymentIntent.decodedObject(fromAPIResponse: response)
```

Comment:
Add comments explaining why you are doing this here and why this isn't part of the JSON